.github/workflows/autocorrect-lint.yml

```diff
@@ -0,0 +1,16 @@
+name: AutoCorrect Lint
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: AutoCorrect
+        uses: huacnlee/autocorrect-action@v1.5.5
```

Comment:
![image](https://user-images.githubusercontent.com/5518/151207608-320b146b-69c6-402f-a6a6-fde679cc752c.png)

`huacnlee/autocorrect-action@v1.5.5` version it works.